db/migrate/20160127193752_create_games.rb

```diff
@@ -0,0 +1,16 @@
+class CreateGames < ActiveRecord::Migration
+  def change
+    create_table :games do |t|
+      t.datetime :start_time, null: false
+      t.belongs_to :week, class_name: 'Week', null: false
```

Comment:
Oh hey look at these options. What do these do in a migration? I'm not sure if they change anything about the database or schema, but I'd love to be proven wrong.